assets/base/i18n/en.json

```diff
@@ -2191,16 +2191,16 @@
   "invite_member.sending": " Sending",
   "invite_member.teamInviteLink": "You can also invite people using the {link}.",
   "katex.error": "Couldn't compile your Latex code. Please review the syntax and try again.",
-  "last_users_message.added_to_channel.type": "were <b>added to the channel</b> by {actor}.",
-  "last_users_message.added_to_team.type": "were <b>added to the team</b> by {actor}.",
+  "last_users_message.added_to_channel.type": "were **added to the channel** by {actor}.",
+  "last_users_message.added_to_team.type": "were **added to the team** by {actor}.",
   "last_users_message.first": "{firstUser} and ",
-  "last_users_message.joined_channel.type": "<b>joined the channel</b>.",
-  "last_users_message.joined_team.type": "<b>joined the team</b>.",
-  "last_users_message.left_channel.type": "<b>left the channel</b>.",
-  "last_users_message.left_team.type": "<b>left the team</b>.",
+  "last_users_message.joined_channel.type": "**joined the channel**.",
+  "last_users_message.joined_team.type": "**joined the team**.",
+  "last_users_message.left_channel.type": "**left the channel**.",
+  "last_users_message.left_team.type": "**left the team**.",
   "last_users_message.others": "{numOthers} others ",
-  "last_users_message.removed_from_channel.type": "were <b>removed from the channel</b>.",
-  "last_users_message.removed_from_team.type": "were <b>removed from the team</b>.",
+  "last_users_message.removed_from_channel.type": "were **removed from the channel**.",
+  "last_users_message.removed_from_team.type": "were **removed from the team**.",
```

Comment:
Changes in above translations were already merged to webapp - https://github.com/mattermost/mattermost-webapp/commit/35af88a26850fd83f38016a4f7cbf9ece982c9ee